### PR TITLE
billing-ut - database initialize properly

### PR DESCRIPTION
**Pull request by ClaudeROSSI**
minor changes - to make it possible to start billing tests

### CODE REVIEW

File: test/api/BillingTest.ts
Comment (jerome-benoit) [inline test/api/BillingTest.ts:117]: Ideally you should test that global.database is falsy, but nobody does that in the unit tests ;)

Comment (ClaudeROSSI) [reply test/api/BillingTest.ts:117]: Would it be possible to get an approval for that  change as is?
I need it in other PR to be able to simply trigger the billing tests. thanks!

Comment (jerome-benoit) [reply test/api/BillingTest.ts:117]: I can approuve it but the merge will be done by Serge when his time permits, unless the rules have changed and now you also have the rights to do merge after one approval by me.

**Review by jerome-benoit — APPROVED** [review]

**Pull request merged**
